common/common-ui/src/main/java/com/duckduckgo/common/ui/view/button/DaxButton.kt

```diff
@@ -63,6 +63,7 @@ open class DaxButton @JvmOverloads constructor(
 
         minHeight = resources.getDimensionPixelSize(Size.dimension(buttonSize))
         setPadding(sidePadding, topPadding, sidePadding, topPadding)
+        setCornerRadiusResource(R.dimen.mediumShapeCornerRadius)
```

Comment:
Can't you change the style instead? /cc @nalcalag

Reply:
I can. Done [here](https://github.com/duckduckgo/Android/pull/6570/commits/7576f433676e3d8bd568fe48225f56a9cbfbd10d).